dts/bindings/sensor/meas,ms5837.yaml

```diff
@@ -6,3 +6,13 @@ description: TE Connectivity MS5837 digital pressure sensor
 compatible: "meas,ms5837"
 
 include: i2c-device.yaml
+
+properties:
+  hardware-type:
+    type: string
+    required: false
+    default: "30BA"
```

Comment:
All use of default must be justified. See https://docs.zephyrproject.org/latest/guides/dts/bindings.html#rules-for-mainline-bindings. My guess is that this should be made `required: true` instead.

Reply:
Updated, thank you